site-permissions/site-permissions-impl/src/main/java/com/duckduckgo/site/permissions/impl/SitePermissionsManagerImpl.kt

```diff
@@ -49,4 +58,12 @@ class SitePermissionsManagerImpl @Inject constructor(
     private fun isPermissionSupported(permission: String): Boolean =
         permission == PermissionRequest.RESOURCE_AUDIO_CAPTURE || permission == PermissionRequest.RESOURCE_VIDEO_CAPTURE
 
+    private fun isHardwareSupported(permission: String): Boolean = when (permission) {
```

Comment:
Now that camera is optional, this avoids showing the camera request dialog on devices we don't support.